docs/azmcp-commands.md

```diff
@@ -123,6 +123,10 @@ azmcp server start \
 # List knowledge indexes in an AI Foundry project
 azmcp foundry knowledge index list --endpoint <endpoint>
 
+# Get knowledge index schema information
+azmcp foundry knowledge index schema --endpoint <endpoint> \
+                                     --index-name <index-name>
```

Comment:
Only have -name in the option name if required to differentiate from another similarly named property

Reply:
Missed in the push. :(